.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 dist: bionic
 node_js:
-  - "10"
+  - "12"
```

Comment:
Not sure if we need another patch release, so I don't know if it is a good idea to do this now.
What travis.yml file will be picked up anyways for the release branch? The one from master or the one from the release branch?

Reply:
Travis always uses job definition from commit being tested. This isn't going to break current release .. not sure about future ones though :)

OTOH, this can wait no rush.

Reply:
While I like having node 12, I dislike not having node 10... In fact I'd rather have node 10, 11 and 12 like in github ci actions...

Reply:
Hmm, agree, changing to 10 & 12.

Reply:
@musculman Node 11 is not supported anymore, not sure if we should waste our time on running CI on this. https://nodejs.org/en/about/releases/